src/Pickles/Pickles.BaseDhtmlFiles/Index.html

```diff
@@ -14,7 +14,7 @@
     <!--[if lt IE 9]>
       <script src="js/html5.js"></script>
     <![endif]-->
-    #### EMBED EXPERIMENTALS ####
```

Comment:
Any special reason why you removed this? It's needed for the experimental features.